pkg/daemon/command/command.go

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2022. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package command
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+)
+
+type Opt = func(cmd *DaemonCommand)
+
+// Define how to build a command line to start a nydusd daemon
+type DaemonCommand struct {
+	// "singleton" "fuse"
+	Mode string `type:"subcommand"`
+	// "blobcache" "fscache" "virtiofs"
+	FscacheDriver  string `type:"param" name:"fscache" default:""`
```

Comment:
Seems we haven't handled the `default` tag, can we remove it first?

Reply:
Yes, lets remove it for now